pubsub/gcppubsub/gcppubsub.go

```diff
@@ -122,6 +123,15 @@ func (o *lazyCredsOpener) defaultConn(ctx context.Context) (*URLOpener, error) {
 			o.err = err
 			return
 		}
+
+		// Connect to the GCP pubsub emulator by overriding the default endpoint
+		// if the 'PUBSUB_EMULATOR_HOST' environment variable is set.
+		// Check https://cloud.google.com/pubsub/docs/emulator for more info.
+		emulatorEndPoint := os.Getenv("PUBSUB_EMULATOR_HOST")
+		if emulatorEndPoint != "" {
+			endPoint = emulatorEndPoint
```

Comment:
Also, please update the package docstring under `URLs` to note that it will connect to the emulator based on `PUBSUB_EMULATOR_HOST`.

Reply:
Ping on this one.

Reply:
Done.